tests/conftest.py

```diff
@@ -11,7 +11,6 @@
     "localstack_snapshot.pytest.snapshot",
     "localstack.testing.pytest.filters",
     "localstack.testing.pytest.fixture_conflicts",
-    "localstack.testing.pytest.detect_thread_leakage",
```

Comment:
LGTM, I think IIRC I had a local branch once where I was disabling this too since its just a bit too noisy and not always relevant. Would love to get the opinion of @dfangl  as well, but this wouldn't block the merge anyway since it can be easily re-enabled if needed.

Reply:
I agree with it being pretty verbose. I'm fine with disabling it, but we might also selectively enable it if the pytest log level is set to DEBUG?
In any case, I'm fine with it being disabled for now.